### DISCUSSION TITLE
RFC: Deprecate `Action`s and introduce `Event`s (everything would stay the same just the naming)

### DISCUSSION THREAD

**Discussion by tomastrajan** [ngrx/platform#3398] [Ideas]
### Information

One of the most important NgRx best practices is to implement actions as `events` instead of `command`

For example the well known "[good action hygiene](https://github.com/timdeschryver/eslint-plugin-ngrx/blob/main/docs/rules/good-action-hygiene.md)" speaks about 

> An action should be a **unique event** in the application. An action should:
>
> * tell where it's dispatched
> * tell what event has occurred

Wouldn't it be beneficial to let go of Redux naming convention and improve NgRx for the future?

### I would be willing to submit a PR to fix this issue

- [ ] Yes
- [X] No

**Comment by brandonroberts** [comment]
I personally would not be in favor of this change as it would cause too much friction in the ecosystem in regards to terminology.
